web/react/stores/post_store.jsx

```diff
@@ -150,6 +150,20 @@ var PostStore = assign({}, EventEmitter.prototype, {
   getDraft: function(channel_id, user_id) {
     return BrowserStore.getItem("draft_" + channel_id + "_" + user_id);
   },
+  storeCommentDraft: function(user_id, parent_post_id, draft) {
```

Comment:
Yah, the store and get functions are there in case we need easier access to the comment drafts, though if user_id is unnecessary, then I'll take them out